app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -122,6 +122,16 @@ def track_document_state(state)
         doc_auth_log.save!
       end
 
+      # FIXME: bikeshed this name
+      def avoid_fsm?
+        IdentityConfig.store.in_person_verify_info_controller_enabled
+      end
+
+      def exit_flow_state_machine
+        flow_session[:flow_path] = @flow.flow_path
+        redirect_to idv_in_person_verify_info_url
```

Comment:
Is this going to dump users to the verify step immediately after completing a step even if they aren't updating the information on that step? If you're going through the flow for the first time, and IdentityConfig.store.in_person_verify_info_controller_enabled = true, they will get redirected at the end of the step?

Reply:
@JackRyan1989 I'm not familiar enough with the IPP flow to answer this with absolute certainty, but I don't think so. `exit_flow_state_machine` is only called in a few spots when information i being updated. I was able to go through the flow without being jumped to anywhere weird.

Reply:
This (or its usages) likely need to differentiate between adding vs updating the information in order to keep the same ordering for the flow. submitting from the State ID step doesn't always lead to the verify info step. Reading this change makes it appear that the Address step (and maybe SSN step) may be skipped in some cases where those are required.

Reply:
@n1zyy Specifically, I would like you to confirm that you completed the following steps during testing:

1. Complete flow up to state ID step
2. Select the option saying that your ID **_does not_** match your address; otherwise complete and submit the form
3. Verify that you are sent to the residential address page, then complete and submit that form
4. Verify that you are at the social security page, then complete and submit that page
5. Check that you're on the "Verify Info" step